styles/index.css

```diff
@@ -0,0 +1,428 @@
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif
+}
+
+/* skeleton */
+header {
+  clear: both;
+  margin-bottom: 0;
+}
+
+main {
+  margin-left: 15rem;
+  margin-right: 15rem;
+}
+
+.maintext {
+  float: left;
+}
+
+aside {
+  float: right;
+}
+
+footer {
+  clear: both;
+  margin-bottom: 0;
+}
+
+/* header setting */
+.header-banner {
+  background-image: linear-gradient(45deg, blue, grey);
```

Comment:
nice work with the gradient! now if the image doesn't load, it'll just render a gradient!